test/cpp/qps/driver.cc

```diff
@@ -198,8 +209,9 @@ std::unique_ptr<ScenarioResult> RunScenario(
     const ServerConfig& initial_server_config, size_t num_servers,
     int warmup_seconds, int benchmark_seconds, int spawn_local_worker_count,
     const grpc::string& qps_server_target_override,
-    const grpc::string& credential_type, bool run_inproc,
-    int32_t median_latency_collection_interval_millis) {
+    const grpc::string& credential_type,
+    std::map<std::string, std::string> per_worker_credential_types,
```

Comment:
Might as well make this const & (and change the QpsDriver() func to store the map in a variable instead of constructing it inline)

Reply:
done